packages/amplify-cli-core/src/utils/windows-utils.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Check if running on Windows OS
+ * @returns true if the CLI is used from Windows machine, false otherwise
+ */
+export const isWindowsPlatform = (): boolean => {
+  return process?.platform?.startsWith('win');
+};
```

Comment:
nit this can just be a const value:
```suggestion
export const isWindowsPlatform = process?.platform?.startsWith('win');
```